backend/controller/lmevaljob_controller.go

```diff
@@ -48,13 +49,25 @@ const (
 	PodImageKey                 = "pod-image"
 	DriverImageKey              = "driver-image"
 	DriverServiceAccountKey     = "driver-serviceaccount"
+	PodCheckingIntervalKey      = "pod-checking-interval"
+	ImagePullPolicyKey          = "image-pull-policy"
 	DefaultPodImage             = "quay.io/yhwang/lm-eval-aas-flask:test"
 	DefaultDriverImage          = "quay.io/yhwang/lm-eval-aas-driver:test"
 	DefaultDriverServiceAccount = "driver"
+	DefaultPodCheckingInterval  = time.Second * 10
+	DefaultImagePullPolicy      = corev1.PullAlways
 )
 
-// EvalJobReconciler reconciles a EvalJob object
-type EvalJobReconciler struct {
+var (
+	pullPolicyMap = map[string]corev1.PullPolicy{
```

Comment:
I'm confused why this is necessary since I would assume [based on the docs](https://pkg.go.dev/k8s.io/api/core/v1#PullPolicy) that `corev1.PullAlways` is exactly equivalent to `"Always"`.

Reply:
I think this is related to your last comment. Since I use `corev1.PullPolicy` for the `ImagePullPolicy` field in the struct `ServiceOptions`, I do need a way to verify if the value in the ConfigMap is valid or not. The underlying corev1.PullPolicy is the string, so I created a map here to have the valid values. I didn't throw any error if the value is invalid, instead, I have `Always` as the default value. Loading the ConfigMap is still in the initialization phase, I definitely can error out and ask users to fix the ConfigMap error. My thinking is that the image-pulling policy is not that critical, so I didn't error out here. We can have documentation saying that
```
The default image-pulling policy is Always, if the value is invalid, Always will be used.
```

But again, I can error out here if you prefer that way.

Reply:
I think my main point was to not duplicate the constants (I'm allergic to multiple sources of truth for the same constant). At a minimum, I think this should be:

```go
pullPolicyMap = map[corev1.PullPolicy]corev1.PullPolicy{
  corev1.PullAlways: corev1.PullAlways,
  ...
}
```

Reply:
sure thing, I just need to construct the `v` into `corev1.PullPolicy()` in this case. updated